main/src/main/java/org/mobilitydata/gtfsvalidator/cli/Main.java

```diff
@@ -171,4 +180,19 @@ private static void exportReport(final NoticeContainer noticeContainer, final Ar
       logger.atSevere().withCause(e).log("Cannot store report files");
     }
   }
+
+  private static void exportNoticeSchema(final Arguments args) {
+    new File(args.getOutputBase()).mkdirs();
+          Gson gson = createGson(args.getPretty());
```

Comment:
This is an experiment that the code will be formatted before merging this PR, right?

Reply:
Yes!